llvm/lib/Transforms/Vectorize/VPlan.cpp

```diff
@@ -1072,9 +1072,13 @@ InstructionCost VPlan::cost(ElementCount VF, VPCostContext &Ctx) {
   // blocks, like the preheader or middle blocks.
```

Comment:
Comment deserves updating?

Reply:
Updated to clarify that other blocks are checked for recipes with invalid costs, thanks.